<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameLauncherClass</class>
 <widget class="QMainWindow" name="GameLauncherClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GameLauncher</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
 font-family:century gothic;
 font-size:14px;
}

#LoginForm
{
 background:url(:/LoginUI/Login_Background.png);
}


QToolButton
{
background:red;
 border-radius:30px;
}

QLabel
{
 color:white;
}

#CreateAccount
{
 color:#908780;
 background:transparent;
}

#Login
{
 color:#908780;
 background:transparent;
}
QLineEdit
{
 background:transparent;
 border:none;
 font-size:24px;
 color:#979797;
}</string>
  </property>
  <widget class="QWidget" name="LoginForm">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>300</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color:#952e25;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>330</width>
      <height>56</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>century gothic</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="CreateAccount">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>300</y>
      <width>56</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover
{
	color:#beb6a5;
}</string>
    </property>
    <property name="text">
     <string>회원가입</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>225</y>
      <width>330</width>
      <height>56</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Login">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>300</y>
      <width>56</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover
{
	color:#beb6a5;
}</string>
    </property>
    <property name="text">
     <string>로그인</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>Login</tabstop>
  <tabstop>CreateAccount</tabstop>
 </tabstops>
 <resources>
  <include location="../Resource/LoginUI.qrc"/>
 </resources>
 <connections/>
</ui>
